haskell.nix/library/.github/workflows/check-nightly.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  schedule:
+    # runs every Monday at 10:00 am UTC
+    - cron: '0 10 * * 1'
+
+jobs:
+  check-nightly:
+    # We use shared runners instead of our selfhosted ones because the stack integrates quite poorly with NixOS
```

Comment:
Hm, in `haskell.nix/library/flake.nix` I see:
```
          # you can specify additional stack resolvers, they will be replaced in stack.yaml
          # resolvers = [ "lts-19.13" ];
```

So our main way of building is still Stack-based. Can't we reuse it and specify `resolvers = [ "nightly" ]`?
I'm fine with not using Nix here, because we don't care much about caching and reproducibility for this specific job, but if it's possible to use the same way of building with low effort, I think it's better to do so.

Reply:
Okay, if it's not that easy, let's keep it as is.